js/webpack.config.js

```diff
@@ -1,22 +1,24 @@
-var loaders = [
-    { test: /\.css$/, loader: "style-loader!css-loader" },
-    { test: /\.less$/, loader: "style-loader!css-loader!less-loader" },
-    { test: /\.json$/, loader: "json-loader" },
-    { test: /\.(jpg|png|gif)$/, loader: "file" },
+const path = require('path');
+
+var rules = [
+    { test: /\.css$/, use: [{loader: "style-loader"}, {loader: "css-loader" }]},
+    { test: /\.less$/, use: [{loader: "style-loader"}, {loader: "css-loader" }, {loader: "less-loader" }]},
+    // { test: /\.json$/, loader: "json-loader" },
```

Comment:
Do we still use the JSON loading in bqplot?

Reply:
I'll have to look into it, but webpack itself failed with this enabled, and everything works without.

Reply:
webpack 4 has a default loader, and the json loader is used to get the version from package.json